src/lib.rs

```diff
@@ -527,6 +527,7 @@ impl Build {
         let host = self.get_host()?;
         let mut cfg = Build::new();
         cfg.flag(flag)
+            .cargo_metadata(self.cargo_metadata)
```

Comment:
It's obvious that the value should be at least inherited. But I'm not sure that fully disabled value will not break anything for consumers.

Reply:
Inherited makes sense to me, since even if it's not disabled, it's results can influence compilation, so we'd want to rerun-on-change for things which might change.